.github/workflows/deploy-master.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-static-page:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: build image
+        run: docker build -t documentation:latest .
+      - name: copy static files locally
+        run: docker run -v ${PWD}:/output documentation:latest cp -r /usr/share/nginx/html /output/_site
```

Comment:
As a next step we could upload the `_site` to the test S3 bucket.
It maybe be simple like this:

```yaml
      - name: upload files to S3 bucket
        run: aws s3 sync _site s3://docs.docker.com-test-us-east-1/ --delete
```